### PR TITLE
[docs][yba] Add master key rotation PLAT-7885

### PR DESCRIPTION
**Pull request by ddhodge**
Add master key rotation

@netlify /preview/yugabyte-platform/security/enable-encryption-at-rest/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md [hunk 1]
```diff
@@ -56,59 +49,85 @@ You can verify that encryption at rest has been successfully configured as follo
...
+
+When restoring an encrypted backup to a universe, Yugabyte Anywhere detects the correct KMS configuration used to encrypt the backup. The KMS configuration must be available in the YugabyteDB Anywhere account.
+
+When restoring your universe data, YugabyteDB Anywhere uses the selected KMS configuration to consume the master key ID and then retrieve the master key value for each key ID in the metadata file. Each of these keys are then sent to the destination universe to augment or build the universe key registry there.
+
+## Rotate the master keys
+
+As part of envelope encryption, the universe keys are protected by master keys. The master key resides in the KMS of your choosing and is used to encrypt and decrypt the universe keys as needed.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md:84]: what is envelope encryption? I got the idea after googling it , but should we provide a link to it or can we assume that a user would be familiar with it?

**Review by aishwarya24 — APPROVED** [review]
LGTM with some comments.

File: docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md
Comment (NValsan) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md:1]: I see a typo for "KMS", shows as "KSM" here "Select your KSM configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-config/)"

File: docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md:84]: added in 1st paragraph on page

**Comment by NValsan** [comment]
had a quick chat with Sahith about the note on AWS KMS in the "Enable ear at cluster creation" section, @ddhodge . This is the current text "If your configuration includes AWS KMS, the following occurs: after the universe has been created with encryption at rest enabled, YugabyteDB Anywhere persists **the ciphertext of the master key** (because AWS does not persist any CMK-generated data keys themselves) and requests the plaintext of the master key from AWS KMS using the KMS configuration whenever it needs to provide the universe key to the master nodes. For more information, see [Create a KMS configuration using AWS KMS](https://deploy-preview-16869--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/security/create-kms-config/aws-kms/).". We only persist the universe key, could we have this updated?

**Pull request merged**